js/src/wp-seo-term-scraper.js

```diff
@@ -52,7 +52,8 @@ var updateAdminBar = require( './ui/adminBar' ).update;
 			snippetTitle: this.getDataFromInput( 'title' ),
 			meta: this.getDataFromInput( 'meta' ),
 			snippetMeta: this.getDataFromInput( 'snippetMeta' ),
-			snippetCite: this.getDataFromInput( 'cite' )
+			snippetCite: this.getDataFromInput( 'cite' ),
+			permalink: this.getDataFromInput( 'baseUrl' ) + this.getDataFromInput( 'url' ) + '/'
```

Comment:
I think it is better to (re)use the variable 'url' instead of calling the 'getDataFromInput' function again.